rdtools/analysis_chains.py

```diff
@@ -242,7 +245,17 @@ def _calc_clearsky_poa(self, times=None, rescale=True, **kwargs):
             **kwargs)
         clearsky_poa = clearsky_poa['poa_global']
 
+        if aggregate:
+            interval_id = pd.Series(range(len(self.poa_global)), index=self.poa_global.index)
+            interval_id = interval_id.reindex(times).bfill()
+            clearsky_poa = clearsky_poa.groupby(interval_id).mean()
+            clearsky_poa.index = self.poa_global.index
```

Comment:
I think we should set the first clearsky value to `NaN` to be strictly correct for right-labeled interval average